src/table/io/FitsSchemaInputMapper.cc

```diff
@@ -405,16 +428,32 @@ class AngleReader : public FitsColumnReader {
         }
     }
 
+    void prepRead(std::size_t firstRow, std::size_t nRows, fits::Fits & fits) override {
+        assert(_key.getElementCount() == 1u);
```

Comment:
We can't have arrays of `Angle`? Pairs of `Angle` could be used for sky coordinates?

Reply:
We now handle struct-like combinations of fields as a layer that sits on top of completely standalone fields instead, the the `FunctorKeys` classes in `lsst/afw/table/aggregates.h`.  That makes working with column arrays easier (i.e. because you can get `ra` and `dec` separately rather than as a 2xN matrix).

There might be a use case for more general arrays of angles, too, but I don't think we've encountered one yet, and until C++17 we're limited to 20 total field types by a hard-to-change upper limit in `boost::variant`.